lib/ex_doc/formatter/html/templates/api_reference_template.eex

```diff
@@ -15,7 +15,7 @@
 
 <%= if nodes_map.tasks != [] do %>
   <section class="details-list">
-    <h2 id="tasks" class="section-heading">Mix Tasks</h2>
```

Comment:
REVIEWER: Not needed, this is injected by `ExDoc.Formatter.HTML.Templates.link_heading/5`